src/actions/index.js

```diff
@@ -155,6 +158,34 @@ export const deviceModaleRevokeClose = () => ({
   type: DEVICES_MODALE_REVOKE_CLOSE
 })
 
+export const installApp = (slug, repourl, isupdate) => {
```

Comment:
Just a matter of syntax's homogeneity: we use camelCase for variable name: `repoURL` and `isUpdate`.